packages/rich-text/src/to-dom.js

```diff
@@ -263,6 +263,10 @@ export function applySelection( selection, current ) {
 		range.setEnd( endContainer, endOffset );
 	}
 
+	if ( document.activeElement !== current ) {
```

Comment:
For what reason do we use `current.ownerDocument` above but `document` directly here?

Reply:
Why does the element need to be focused? (Perhaps as an inline comment)